specification/operationalinsights/resource-manager/readme.md

```diff
@@ -39,7 +39,6 @@ These settings apply only when `--tag=package-2015-11-preview` is specified on t
 ``` yaml $(tag) == 'package-2015-11-preview'
 input-file:
 - Microsoft.OperationalInsights/preview/2015-11-01-preview/OperationalInsights.json
-- Microsoft.OperationalInsights/stable/2015-03-20/OperationalInsights.json

```

Comment:
Just to confirm one last time before we merge: Was the original reference to this file a typo?

Reply:
Yes, I believe so